certora/specs/Reverts.spec

```diff
@@ -0,0 +1,253 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+import "ConsistentInstantiation.spec";
+
+using Morpho as MORPHO;
+
+methods {
+    function _.position(PreLiquidation.Id, address) external => DISPATCHER(true);
+    function _.accrueInterest(PreLiquidation.MarketParams) external => DISPATCHER(true);
+    function _.borrowRate(PreLiquidation.MarketParams, PreLiquidation.Id) external => HAVOC_ECF;
```

Comment:
I think this is useless, because HAVOC_ECF is already the AUTO summary (if you don't specify anything)

Reply:
May have been forgotten from the `marketExists` rule.

Reply:
Removed in https://github.com/morpho-org/pre-liquidation/pull/37/commits/ebdaff6e0cdf9a9030237d530d03bf9708364178.